src/pages/transactions/[id].js

```diff
@@ -16,18 +24,23 @@ export default function TransactionDetails() {
   return (
     <div>
       <div>
-        <Link href={"/"}><ChevronLeft/> Back</Link>
+        <Link href={"/"}>
+          <ChevronLeft /> Back
+        </Link>
         <h1>{transaction.category}</h1>
+        <button onClick={onConfirmDelete}>
+          <DeleteBin />
+        </button>
       </div>
       <div>
         <h2>Amount</h2>
-        <div>{transaction.amount.toFixed(2)} {transaction.currency}</div>
+        <div>
+          {transaction.amount.toFixed(2)} {transaction.currency}
+        </div>
       </div>
       <div>
```

Comment:
Minor comment: you tend to use a whole lot of div/s. Try to avoid, since this makes the code difficult to parse (for me, at least).

Reply:
converted to a list